dbt/README.md

```diff
@@ -0,0 +1,79 @@
+
+## Structure
+
+- [`dbt_project.yml`](command:_github.copilot.openRelativePath?%5B%7B%22scheme%22%3A%22file%22%2C%22authority%22%3A%22%22%2C%22path%22%3A%22%2FUsers%2Fgmendoza%2Fsource%2Fcivis-dbt-docker%2Fplatform-code-examples%2Fdbt%2Fdbt_project.yml%22%2C%22query%22%3A%22%22%2C%22fragment%22%3A%22%22%7D%5D "/Users/gmendoza/source/civis-dbt-docker/platform-code-examples/dbt/dbt_project.yml"): Main configuration file for the dbt project.
+- [`profiles.yml`](command:_github.copilot.openRelativePath?%5B%7B%22scheme%22%3A%22file%22%2C%22authority%22%3A%22%22%2C%22path%22%3A%22%2FUsers%2Fgmendoza%2Fsource%2Fcivis-dbt-docker%2Fplatform-code-examples%2Fdbt%2Fprofiles.yml%22%2C%22query%22%3A%22%22%2C%22fragment%22%3A%22%22%7D%5D "/Users/gmendoza/source/civis-dbt-docker/platform-code-examples/dbt/profiles.yml"): Profiles configuration for dbt, specifying different environments and their settings.
+- `models/`: Directory containing SQL models for dbt.
+  - `docs.md`: Documentation for the models.
+  - [`overview.md`](command:_github.copilot.openRelativePath?%5B%7B%22scheme%22%3A%22file%22%2C%22authority%22%3A%22%22%2C%22path%22%3A%22%2FUsers%2Fgmendoza%2Fsource%2Fcivis-dbt-docker%2Fplatform-code-examples%2Fdbt%2Fmodels%2Foverview.md%22%2C%22query%22%3A%22%22%2C%22fragment%22%3A%22%22%7D%5D "/Users/gmendoza/source/civis-dbt-docker/platform-code-examples/dbt/models/overview.md"): Overview of the dbt project.
+  - `schema.yml`: Schema configuration for the models.
+  - `staging/`: Directory containing staging models.
+    - `stg_customers.sql`: SQL model for staging customers.
+    - `stg_orders.sql`: SQL model for staging orders.
+    - `stg_payments.sql`: SQL model for staging payments.
+- `seeds/`: Directory containing seed files for dbt.
+
+## Configuration
+
+### dbt_project.yml
+
+The [`dbt_project.yml`](command:_github.copilot.openRelativePath?%5B%7B%22scheme%22%3A%22file%22%2C%22authority%22%3A%22%22%2C%22path%22%3A%22%2FUsers%2Fgmendoza%2Fsource%2Fcivis-dbt-docker%2Fplatform-code-examples%2Fdbt%2Fdbt_project.yml%22%2C%22query%22%3A%22%22%2C%22fragment%22%3A%22%22%7D%5D "/Users/gmendoza/source/civis-dbt-docker/platform-code-examples/dbt/dbt_project.yml") file contains the main configuration for the dbt project, including the project name, version, paths to models, seeds, and other configurations.
+
+### profiles.yml
+
+The [`profiles.yml`](command:_github.copilot.openRelativePath?%5B%7B%22scheme%22%3A%22file%22%2C%22authority%22%3A%22%22%2C%22path%22%3A%22%2FUsers%2Fgmendoza%2Fsource%2Fcivis-dbt-docker%2Fplatform-code-examples%2Fdbt%2Fprofiles.yml%22%2C%22query%22%3A%22%22%2C%22fragment%22%3A%22%22%7D%5D "/Users/gmendoza/source/civis-dbt-docker/platform-code-examples/dbt/profiles.yml") file contains the profiles configuration for dbt, specifying different environments (e.g., development, production) and their settings, such as database credentials and connection details.
```

Comment:
these links look a little wonky - is there a way to define them without including your local environment's path? `Users/gmendoza/source/civis-dbt-docker/platform-code-examples/`

Reply:
oh shoot. yeah def